documentation/en/api-v1-unstable-methods.md

```diff
@@ -3383,6 +3387,61 @@ Response:
 }
 ```
 
+## Get
+
+
+### GetActorEvents
+Actor events
+
+
+Perms: read
+
+Inputs:
+```json
+[
+  {
+    "address": [
+      "f01234"
+    ],
+    "fields": {
+      "abc": [
+        {
+          "codec": 81,
+          "value": "ZGF0YQ=="
+        }
+      ]
+    },
+    "minEpoch": 2301220,
+    "maxEpoch": 2301220
+  }
+]
+```
+
+Response:
+```json
+[
+  {
+    "Entries": [
+      {
+        "Flags": 7,
+        "Key": "string value",
+        "Codec": 42,
+        "Value": "Ynl0ZSBhcnJheQ=="
```

Comment:
It would be nice if this documentation explained how to interpret Codec and how to parse Value for common events that clients may want to observe.

For example:
- Link to relevant specs describing known events and their key/codec/value.
- Assuming this example is based on such an event, can you improve this example to link to that event docs and show what we get after parsing the value `Ynl0ZSBhcnJheQ==` using the code `42`.

Reply:
Hey yes,

We will document this once we have rough consensus on the API from other partners/code reviews from devs.